python-frontend/typeshed_serializer/requirements.txt

```diff
@@ -6,3 +6,36 @@ toml==0.10.2
 typing-extensions==3.10.0.0
 pytest==7.2.0
 pytest-cov==4.0.0
+
+# Stub dependencies
+atomicwrites==1.4.1
+backports-abc==0.5
+certifi==2023.5.7
+characteristic==14.3.0
+click==8.1.3
+cryptography==36.0.1
+dataclasses==0.6
+enum34==1.1.10
+filelock==3.4.0
+first==2.0.2
+Flask==2.3.2
+freezegun==1.2.2
+frozenlist==1.3.3
+futures==3.0.5
+geoip2==4.7.0
+ipaddress==1.0.23
+itsdangerous==2.1.2
+Jinja2==3.1.2
+kazoo==2.9.0
+MarkupSafe==2.1.2
+maxminddb==2.3.0
```

Comment:
The package to import nmap is called python-nmap==0.7.1

Reply:
Somehow, adding this didn't change the generated protobuf, so I removed `nmap` altogether.